tests/test_examples.py

```diff
@@ -1261,3 +1261,30 @@ def test_v1_0_arg_empty_prefix_separate_false() -> None:
     )
     assert "completed success" in stderr
     assert error_code == 0
+
+
+def test_scatter_output_filenames(tmpdir: py.path.local) -> None:
+    """Check that when a scatter step produces same named output the final output is renamed correctly."""
```

Comment:
```suggestion
    """If a scatter step produces identically named output then confirm that the final output is renamed correctly."""
```
Small wording tweak; thanks for this test!